.github/workflows/spack_cpu_build.yaml

```diff
@@ -18,35 +26,88 @@ jobs:
         # - exago@develop~mpi~ipopt~hiop~python~raja
           # See #44 - +mpi~python should fail if no python in system,
           #                       but the runner happens to have one...
-          - exago@develop+mpi~ipopt+hiop~python~raja ^openmpi
+          - exago@1.6.0+mpi~ipopt+hiop~python~raja ^openmpi
```

Comment:
Why is this a version number instead of develop? If this is running on MRs, don't we want it to grab the current version?